homeassistant/components/cover/myq.py

```diff
@@ -28,55 +27,47 @@
     'opening': STATE_OPENING
 }
 
-NOTIFICATION_ID = 'myq_notification'
-NOTIFICATION_TITLE = 'MyQ Cover Setup'
-
 COVER_SCHEMA = vol.Schema({
```

Comment:
This should be renamed to `PLATFORM_SCHEMA` and extend the base component platform schema.